{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/app.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+import session_csrf
+
+
+class BaseAppConfig(AppConfig):
+    name = '{{ cookiecutter.project_name }}.base'
+
+    def ready(self):
+        session_csrf.monkeypatch()
```

Comment:
This reads as if the whole point of this AppConfig class is specifically only for the sake of `session_csrf`.
Perhaps here would be a good idea to write a more verbose comment.